prow/plugins/docs-no-retest/docs-no-retest_test.go

```diff
@@ -0,0 +1,350 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package docsnoretest
+
+import (
+	"errors"
+	"testing"
+
+	"k8s.io/test-infra/prow/github"
+)
+
+type ghc struct {
+	*testing.T
+	labels    map[string]bool
+	prChanges []github.PullRequestChange
+
+	addLabelErr, removeLabelErr, getIssueLabelsErr, getPullRequestChangesErr error
+}
+
+func (c *ghc) AddLabel(_, _ string, _ int, targetLabel string) error {
+	c.T.Logf("AddLabel: %s", targetLabel)
+	c.labels[targetLabel] = true
+
+	return c.addLabelErr
+}
+
+func (c *ghc) RemoveLabel(_, _ string, _ int, targetLabel string) error {
+	c.T.Logf("RemoveLabel: %s", targetLabel)
+	for label := range c.labels {
+		if label == targetLabel {
+			delete(c.labels, label)
```

Comment:
You're doing a deletion here so you don't really need the `bool`, can just use `map[string]interface{}` (the k8s `set.String` utility)